fluentd/lib/in_tail/write_watcher.rb

```diff
@@ -0,0 +1,72 @@
+#
```

Comment:
This file should be either removed entirely, appear as a patch to vendored_src, are a new file to vendored_src since you cant patch it

Reply:
All the files except write_watcher.rb should be patches to existing vendored files. They were renamed when I thought we wer creating a separate plugin, if we're patching we should have the minimum diff from the vendored files.

write_watcher is misnamed, all it contains now is mertric exporter code. Probably it should be merged into the vendored fluentd exporter, wherever that is.

Reply:
@jcantrill  i would like to know where we can keep this file version controlled as a single piece of truth to all changes that are used to create patch over vendored_src baseline implementation of this plugin.

Reply:
@pmoogi-redhat the patch file goes in the /fluentd directory, patches are installed during the image build by https://github.com/openshift/origin-aggregated-logging/blob/master/fluentd/install-gems.sh 
See the comment at the top about naming the patch file.